koa-crud/test/integration/models/member.spec.ts

```diff
@@ -94,4 +94,17 @@ describe('Member Models', () => {
       });
     });
   });
+
+  describe('Find Members', () => {
```

Comment:
You don't need to add test for `find`, `findOne` and `exists` when testing the Models

Reply:
noted sir